### ISSUE TITLE
Can't save report when user coordinates are too big

### ISSUE THREAD

**Issue by piotrskibapl** [MrTigreroux/TigerReports#97]
I found an issue with reporting players if you are far away from the world spawn:
```
[19:35:08] [Server thread/INFO]: username1 issued server command: /report username2 test
[19:35:09] [Craft Scheduler Thread - 4388/ERROR]: [TigerReports] An error has occurred with the database:
com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'reporter_location' at row 1
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:104) ~[patched_1.16.5.jar:git-Paper-794]
at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953) ~[patched_1.16.5.jar:git-Paper-794]
at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1092) ~[patched_1.16.5.jar:git-Paper-794]
at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1040) ~[patched_1.16.5.jar:git-Paper-794]
at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1347) ~[patched_1.16.5.jar:git-Paper-794]
at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1025) ~[patched_1.16.5.jar:git-Paper-794]
at fr.mrtigreroux.tigerreports.data.database.Database.insert(Database.java:107) ~[?:?]
at fr.mrtigreroux.tigerreports.commands.ReportCommand$2$2$1.run(ReportCommand.java:260) ~[?:?]
at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:101) ~[patched_1.16.5.jar:git-Paper-794]
at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54) ~[patched_1.16.5.jar:git-Paper-794]
at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.16.5.jar:git-Paper-794]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[?:1.8.0_121]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[?:1.8.0_121]
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
```

The exact reporter location in this case was:
x: -9595.809
y: 100.000
z: 11843.694
dimension: nether

TigerReports version: 4.13
Server version: 1.16.5 (git-Paper-794)

**Comment by MrTigreroux** [comment]
Hello, effectively I didn't thought about that. But you can edit your database running this query:
ALTER TABLE tigerreports_reports MODIFY reporter_location (1000) NULL;
ALTER TABLE tigerreports_reports MODIFY reported_location (1000) NULL;
See [this tutorial](https://github.com/MrTigreroux/TigerReports/wiki/Run-a-SQL-query-on-a-MySQL-database) if you don't know how to execute such queries.

**Comment by MrTigreroux** [comment]
Fixed in new [version 4.13.1](https://www.spigotmc.org/resources/tigerreports.25773/update?update=441055).
Thank you again for reporting it.
